apache/trafficcontrol
# TP: adds the ability to add/remove required server capabilities from DS

## What does this PR (Pull Request) do?
This PR provides users with the ability to:
- assign/remove required server capabilities to/from a delivery service
- view all delivery services that have a specific required server capability and, if needed, remove a delivery service from the capability.

- [x] This PR is not related to any Issue but is related to this [blueprint](https://github.com/apache/trafficcontrol/blob/master/blueprints/server-capabilitites.md)

## Which Traffic Control components are affected by this PR?

- Documentation
- Traffic Portal

## What is the best way to verify this PR?
- Launch TP
- ensure that server capabilities exist: tp.domain.com/#!/server-capabilities (if not, add one or two)
- navigate to any delivery service: tp.domain.com/#!/delivery-services/{id}
- click on More > Manage Server Capabilities
- add/remove capabilities to/from the delivery service but leave at least one capability on the delivery service.
- navigate back to a capability you left on the delivery service - tp.domain.com/#!/server-capabilities/{capability name}
- click 'view delivery services' to see the delivery services that utilize the capability and remove a delivery service from the capability.

## The following criteria are ALL met by this PR



## Additional Information
